api/staged_products_property_collection_helpers.go

```diff
@@ -0,0 +1,142 @@
+package api
```

Comment:
So I've been tossing back and forth between how I feel about the 'helpers' method. Since this is just for collections, I think I'd prefer this to be named `collections_service.go`.

Reply:
Happy to get rid of smelly `helpers` name.

However; I'm not sure `collections_service` is better; for two reasons:

1.  It implies a more `generic` set of functionality than is implemented - which is very specific to the (complicated) way the `staged_products` API works.  I think we should keep a name that includes this specificity (eg: `staged_products_property_collection_<something>`
2.  It's quite different to all the other `_service` files in that it doesn't make any network calls.

What about `staged_products_property_collection_GUID_association` or `staged_products_property_collection_GUID_assigner`?

Reply:
fair points.
optimally, I'd love to see all of those things just in the `staged_config_service`, but recognize that this might be a bit of work and a little harder to test the more intricate functionality. These functions are _only_ used in `staged_config`, so even if we rename the file, it still feels a little weird. 

given the choice, I really like `staged_products_property_collection_guid_assigner` name (lowercase instead of uppercase guid in the name)

Reply:
Renamed to `staged_products_property_collection_guid_assigner` by e9c69d6

Reply:
Renamed to `staged_products_property_collection_guid_assigner` by e9c69d6